AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -3488,6 +3488,20 @@ - (UIView *)preferredFocusedView
 
 #pragma mark - Deprecated
 
+- (NSString *)name
+{
+  ASDN::MutexLocker l(__instanceLock__);
```

Comment:
rewrite this to do return self.debugName in one line - then the locking can be done in debugName accessor